### PR TITLE
docs: Clarification on which command to use on which Ubuntu version.

### PR DESCRIPTION
**Pull request by kkovacs**
### SUMMARY
Clarification on which command to use on which Ubuntu version.

### ADDITIONAL INFORMATION
None.

### CODE REVIEW

File: docs/docs/installation/pypi.mdx [hunk 1]
```diff
@@ -22,13 +22,13 @@ level dependencies.
...
 
-The following command will ensure that the required dependencies are installed:
+In Ubuntu **before 20.04** the following command will ensure that the required dependencies are installed:
 
 ```bash
 sudo apt-get install build-essential libssl-dev libffi-dev python-dev python-pip libsasl2-dev libldap2-dev default-libmysqlclient-dev
 ```
 
-In Ubuntu 20.04 the following command will ensure that the required dependencies are installed:
+In Ubuntu **20.04 and 22.04** the following command will ensure that the required dependencies are installed:
```
Comment (michael-s-molina) [inline docs/docs/installation/pypi.mdx:31]: What happens if it's a Ubuntu version that is not in the range? Maybe add this additional context.

**Review by sfirke — CHANGES_REQUESTED** [review]
Minor change needed but thanks for clarifying this, it will be an improvement!

File: docs/docs/installation/pypi.mdx (hunk 1 above)
Comment (kkovacs) [reply docs/docs/installation/pypi.mdx]: Sadly, apparently superset can't be installed yet on Ubuntu 24.04. TLDR: the included `numpy` is too old.

Longer story:

- Ubuntu 24.04 has `python3.12`.
- `numpy` below 1.26.4 is incompatible with `python3.12`.
- Superset requires `numpy==1.23.5`.

Apparently, there was an issue about this earlier, but it was closed without resolution: https://github.com/apache/superset/issues/27585

**Review by sfirke — APPROVED** [review]

**Comment by sfirke** [comment]
Thanks for this contribution @kkovacs !

**Pull request merged**